maikthomas/sframe
# Add option to display failed decrypted frames in receiver

To test:
- checkout branch and `npm i`
- run with `npm run example` 
- in localhost:9000 see video it send and received. Change the password in either sender or receiver. See garbled video in receiver

- change line example/e2ee.js#L82 to `false` and check that video is no longer garbled, it just stops.
